app/src/main/java/org/oppia/android/app/mydownloads/downloads/DownloadsAccessDialogFragment.kt

```diff
@@ -0,0 +1,50 @@
+package org.oppia.android.app.mydownloads.downloads
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import org.oppia.android.app.fragment.InjectableDialogFragment
+import javax.inject.Inject
+
+const val ADMIN_PIN_SAVED_KEY = "DownloadsFragment.admin_pin"
+const val INTERNAL_PROFILE_ID_SAVED_KEY = "DownloadsFragment.internal_profile_id"
+const val ALLOW_DOWNLOAD_ACCESS_SAVED_KEY = "DownloadsFragment.allow_download_access"
```

Comment:
Can we make all these private?

Actually I think the best way is to shift them to companion object and make them internal and access them via this `DownloadsAccessDialogFragment . ADMIN_PIN_SAVED_KEY ` wherever required.

Reply:
done